sound/sound_library.h

```diff
@@ -143,6 +143,7 @@ class SoundLibrary {
   void SayAuto() { Play("mauto.wav"); }
   void SayBaseColor() { Play("mbase.wav"); }
   void SayBatteryLevel() { Play("mbatt.wav"); }
+  void SayTheBatteryLevelIs() { Play("battlevl.wav"); }
```

Comment:
I think it's either A or D. However if you guys disagree on what it what the best way is, then it's E; add both functions and both sounds, then use whichever one you prefer.

Reply:
It really really doesn't matter. I'm not trying to rock the boat, just trying to contribute. If it's harmless enough, and since the voicepacks support it, is giving end users another option a bad thing? How about E.

Reply:
E works for me